tools/devops/automation/templates/build/build.yml

```diff
@@ -288,11 +288,6 @@ steps:
       CONFIGURE_FLAGS="--enable-xamarin"
     fi
 
-    if [[ "$EnableDotNet" == "True" ]]; then
```

Comment:
This will need to be reverted like the other changes. not needed for the bump, but why build it if this branch is just for classic?